cfme/tests/middleware/test_middleware_deployment.py

```diff
@@ -132,3 +149,228 @@ def test_tags(provider):
     assert len(deps_db) > 0, "There is no deployment(s) available in UI"
     deployment = get_random_list(deps_db, 1)[0]
     deployment.validate_tags(tags=tags)
+
+
+@pytest.mark.parametrize("archive", [RESOURCE_WAR_PATH, RESOURCE_JAR_PATH, RESOURCE_EAR_PATH])
+def test_deploy(provider, archive):
+    """Tests Deployment of provided archive into EAP7 server
+
+    Steps:
+        * Get servers list from UI
+        * Chooses JBoss EAP server from list
+        * Invokes 'Add Deployment' toolbar operation
+        * Selects "war" file to upload.
+        * Chose random Runtime Name.
+        * Checks that notification message is shown.
+        * Refreshes the provider.
+        * Verifies that deployment is shown in list and is Enabled.
+        * Selects deployment to show the details.
+        * Verified details properties.
+    """
+    server = get_eap_server(provider)
+    deploy(provider, server, archive)
+    # disable for now as not stable test because of performance issue #10138
+    # _check_deployment_enabled(provider, server, runtime_name)
+    # deployment = _get_deployment_from_list(provider, server, runtime_name)
+    # deployment.validate_properties()
+
+
+@pytest.mark.meta(blockers=['GH#ManageIQ/manageiq:10138'])
+@pytest.mark.uncollect
+@pytest.mark.parametrize("archive", [RESOURCE_WAR_PATH, RESOURCE_JAR_PATH, RESOURCE_EAR_PATH])
```

Comment:
parametrizing by full paths seems not a good option since if it gets run on a slightly different setup, the params change even though the tests is the same. I would rather recommend passing the filenames here and joining the path inside the test, that leads to more stable IDs.